com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstanceHelpers.cs

```diff
@@ -114,10 +114,18 @@ public static void Destroy()
             // Shutdown the server which forces clients to disconnect
             foreach (var networkManager in s_NetworkManagerInstances)
             {
-                if (networkManager.IsServer)
+                if (networkManager.IsHost)
                 {
                     networkManager.StopHost();
                 }
+                else if (networkManager.IsServer)
```

Comment:
Interesting...I would probably put this in its own PR to fix missing stuff in MultiInstanceHelper